update-deps.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dep ensure
```

Comment:
In other repos we've deleted `BUILD{.bazel}` files so that gazelle is solely responsible for generating them, and `*_test.go` to reduce vendored code. Worth doing that here too?

```
rm -rf $(find vendor/ -name 'BUILD')
rm -rf $(find vendor/ -name 'BUILD.bazel')
rm -rf $(find vendor/ -name '*_test.go')
```